assets/js/pg-product-simulator.js

```diff
@@ -0,0 +1,115 @@
+/** global console, simulatorData  */
```

Comment:
If you add this file, you need to remove the code from the template

Reply:
I will remove the template product_simulator.php  because it's not used anymore